src/common/options.rs

```diff
@@ -1887,7 +1887,12 @@ impl OptionsBuilder {
     ///
     /// See the [Deepgram Callback Method feature docs][docs] for more info.
     ///
+    /// Note that modifying the callback method is only available for pre-recorded audio.
+    /// See the [Deepgram Callback feature docs for streaming][streaming-docs] for details
+    /// on streaming callbacks.
```

Comment:
Why is that?  At least when the provided URL is a non-websocket (`http` or `https`) connection, couldn't we use any method the user wants?  Seems like that would be a pretty easy and obvious feature lift in stem.

Reply:
I agree it seems like a "pretty easy and obvious feature lift" in the API. I can only confirm that this is in fact the current behavior, and changing that seems out of scope for this specific PR, so I didn't tackle that yet.